### PR TITLE
fix the access_token loading issue in UI

### PR DESCRIPTION
**Pull request by sanethmaduranga**
## Purpose
This commit will fix the issue of access token loading in the UI

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag [hunk 1]
```diff
@@ -242,27 +242,28 @@ var loadTokens = function () {
...
-	       } else if (selected_type == "sandbox") {
-		     if (result.apikeys.length > 1) {
-			token = result.apikeys[1].SANDBOX_KEY;
-		        $("#access_token").val(token);
-		     } else {
-			$("#access_token").val("");
-		     }
-	       }
-	       if(token && token.trim() != "") {
+            if (selected_type == "production") {
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag]: looks like the alignment is not correct. please verify.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag [hunk 2]
```diff
@@ -242,27 +242,28 @@ var loadTokens = function () {
...
+                for (var key : result.apikeys) {
+                    token = key.PRODUCTION_KEY;
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag]: What happens when the token is undefined? Do we get a warning from the UI?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag (hunk 1 above)
Comment (sanethmaduranga) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag]: Fixed in commit 7d9d783ff33a7893e835d34e0098ac5da6afef37.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag (hunk 2 above)
Comment (sanethmaduranga) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag]: Fixed in commit 7d9d783ff33a7893e835d34e0098ac5da6afef37.

**Comment by bhathiya** [comment]
@isharac please review and merge.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag [hunk 3]
```diff
@@ -242,27 +242,28 @@ var loadTokens = function () {
...
+                    if (typeof token != 'undefined' && token){
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag]: formatting issue. a space is required.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag [hunk 4]
```diff
@@ -242,27 +242,28 @@ var loadTokens = function () {
...
+                        break;
+                    }
+                }
+                $("#access_token").val(token);
+            } else if (selected_type == "sandbox") {
+                for (var key : result.apikeys) {
+                    token = key.SANDBOX_KEY;
+                    if (typeof token != 'undefined' && token){
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag]: formatting issue. a space is required.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag [hunk 5]
```diff
@@ -242,27 +242,28 @@ var loadTokens = function () {
...
+                        break;
+                    }
+                }
+                $("#access_token").val(token);
+            }
+	        if(token && token.trim() != "") {
                 swaggerUi.api.clientAuthorizations.add("default", new SwaggerClient.ApiKeyAuthorization("Authorization", "Bearer "+ token, "header"));
-	       }else{
+	        }else{
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/api/swagger/template.jag]: formatting issue. space is required.

**Pull request merged**